lib/node_modules/@stdlib/blas/ext/base/ndarray/ssorthp/docs/types/index.d.ts

```diff
@@ -0,0 +1,55 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/// <reference types="@stdlib/types"/>
+
+import { typedndarray } from '@stdlib/types/ndarray';
+
+/**
+* Sorts a one-dimensional single-precision floating-point ndarray using heapsort.
+*
+* @param arrays - array-like object containing a one-dimensional input ndarray and a zero-dimensional ndarray specifying the sort order
+* @returns input ndarray
+*
+* @example
+* var Float32Array = require( '@stdlib/array/float32' );
+* var ndarray2array = require( '@stdlib/ndarray/base/to-array' );
+* var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+* var ndarray = require( '@stdlib/ndarray/base/ctor' );
+*
+* var xbuf = new Float32Array( [ 1.0, -2.0, 3.0, -4.0 ] );
+* var x = new ndarray( 'float32', xbuf, [ 4 ], [ 1 ], 0, 'row-major' );
+*
+* var ord = scalar2ndarray( 1.0, {
+*    'dtype': 'generic'
+* });
+*
+* var out = ssorthp( [ x, ord ] );
+* // returns <ndarray>
+*
+* var arr = ndarra2array( out );
+* // returns [ -4.0, -2.0, 1.0, 3.0 ]
+*/
+declare function ssorthp<T = unknown>( arrays: [ typedndarray<T>, typedndarray<number> ] ): typedndarray<T>;
```

Comment:
This isn't correct. The input ndarray needs to be a `float32ndarray`.

Reply:
And similarly, the output ndarray should be a `float32ndarray`. You'll need to update `test.ts`, as well.